example-app/src/main/java/com/adyen/checkout/example/ui/main/ComponentItem.kt

```diff
@@ -0,0 +1,13 @@
+package com.adyen.checkout.example.ui.main
+
+internal sealed class ComponentItem {
+
+    abstract val text: String
+
+    data class Title(override val text: String) : ComponentItem()
+
+    sealed class Entry(override val text: String) : ComponentItem() {
+        object DropIn : Entry("Start")
```

Comment:
Minor: extract these into string resources

Reply:
Will do, hoped this would be okay in the example app 😬